bpf/bpf_overlay.c

```diff
@@ -113,7 +115,6 @@ static __always_inline int handle_ipv6(struct __ctx_buff *ctx,
 	if (1) {
```

Comment:
Not related to that PR, but I've been curious: Do you know what those `if (1)` are for?

Reply:
My assumption is to get a new scope for declaring vars, here host_mac and router_mac. Could also have been plain `{ ... }`to make it more clear.